homeassistant/components/device_tracker/traccar.py

```diff
@@ -52,16 +54,21 @@
 
     api = API(hass.loop, session, config[CONF_USERNAME], config[CONF_PASSWORD],
               config[CONF_HOST], config[CONF_PORT], config[CONF_SSL])
+
     scanner = TraccarScanner(
-        api, hass, async_see, config.get(CONF_SCAN_INTERVAL, SCAN_INTERVAL))
+        api, hass, async_see,
+        config.get(CONF_SCAN_INTERVAL, SCAN_INTERVAL),
+        config.get(CONF_MONITORED_CONDITIONS, []))
```

Comment:
voluptuous is handling the default already.

Reply:
So should I remove the default?

Reply:
Done, thanks :)